src/material-experimental/mdc-slider/slider.ts

```diff
@@ -709,14 +705,34 @@ export class MatSlider extends _MatSliderMixinBase
       : this._foundation.setValue(value);
   }
 
+  _setDisabled(value: boolean, updateControlValueAccessor: boolean = true) {
```

Comment:
We have a section in the coding standards about avoiding boolean arguments like this. Maybe this method should be split into two?

Reply:
I moved the custom logic into a separate function I named `_updateInputsDisabledState`